FoldingTabBar/Controller/FoldingTabBarController/YALFoldingTabBarController.m

```diff
@@ -6,11 +6,11 @@
 #import "YALTabBarItem.h"
 
 //protocol
```

Comment:
Since you deleted YALTabBarInteracting protocol could you plz delete Protocols folder from the project and this legacy comment as well. Thanks.